kolibri/content/utils/paths.py

```diff
@@ -16,6 +16,8 @@
 # set of file extensions that should be considered zip files and allow access to internal files
 POSSIBLE_ZIPPED_FILE_EXTENSIONS = set([".perseus", ".zip", ".epub", ".epub3"])
 
+def get_content_file_name(obj):
+    return '{checksum}.{extension}'.format(checksum=obj.id, extension=obj.extension)
```

Comment:
No, that would mess us up. Not sure how to deal with it, except to try to ban it.